notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -32,18 +32,18 @@
         "<table align=\"left\">\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    ! gcloud ai endpoints delete $ENDPOINT_ID --quiet --region $REGION
</pre><p>Instead of gcloud, just use <code>endpoint.delete</code>.You can add the force parameter if required.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
Changing to endpoint.delete